libdnf.spec

```diff
@@ -13,7 +13,7 @@
 %global oldname libhif
 
 Name:           libdnf
-Version:        0.7.0
+Version:        0.7.1
```

Comment:
@j-mracek Things that use this new change can have their pre-release requirement bumped. For example, in the DNF package, you can do `python3-hawkey > 0.7.0-0.7`. This is what I do in Mageia as you guys break the ABI/API. See for example [how I do it in PackageKit](http://svnweb.mageia.org/packages/cauldron/packagekit/current/SPECS/packagekit.spec?view=markup).